Modules/_sqlite/connection.c

```diff
@@ -2173,6 +2173,45 @@ pysqlite_connection_create_collation_impl(pysqlite_Connection *self,
     Py_RETURN_NONE;
 }
 
+/*[clinic input]
+_sqlite3.Connection.set_file_control as pysqlite_connection_set_file_control
```

Comment:
`sqlite3_file_control` is an API for querying and setting values. `set_file_control` is misleading, since the API can also be used to query values. See similar API wrappers for `sqlite3_db_config` and `sqlite3_limit` in connection.c. How about making this a set/get pair: `setfilecontrol`/`getfilecontrol`? If we're adding it as a single API, let's go with `filecontrol`.